changes-set.txt

```diff
@@ -27,6 +27,9 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+31-Jul-23 sbcgfi    [same]      moved from GM's mathbox to main set.mm
```

Comment:
Why did you move them to main?

Reply:
I had used them in my mathbox in my first commit, but it seems that the corresponding proofs are removed/have changed, so that they are not used anymore.